src/storybook/components/usage-guidelines/usage-guideline.scss

```diff
@@ -4,16 +4,23 @@
 .monday-storybook-usage-guidelines{
   margin-top: $spacing-between-section-items;
   &_guideline {
-    margin: $spacing-medium 0;
+    color: var(--primary-text-color);
+    margin: var(--spacing-medium) var(--spacing-small);
     display: flex;
     align-items: center;
+    font-size: 16px;
+    line-height: 21px;
     font-weight: 300;
     &:first-child {
       margin-top: 0;
     }
   }
 
   &_icon {
-    margin-right: $spacing-small;
+    font-size: 16px;
```

Comment:
same note about the line height. is this intentional?

Reply:
the font was a bit bigger (it looked odd)